DevelopContainer/Dockerfile

```diff
@@ -1,5 +1,5 @@
 
-ARG COMPILER="gcc"
+ARG compiler "gcc"
```

Comment:
Why do you have to change this to lower case? The variable was used as `${COMPILER}` and worked fine , didnt it?

Reply:
Ah the comment 
```
#docker build --squash -t ikarusproject/ikarus-dev-gcc:latest --build-arg compiler="gcc" --build-arg ccompiler="gcc-12" --build-arg cppcompiler="g++-12"  . &> build.log
#docker build --squash -t ikarusproject/ikarus-dev-clang:latest --build-arg compiler="clang" --build-arg ccompiler="clang-16" --build-arg cppcompiler="clang++-16"  . &> build.log
```
is wrong. We can go with lowercase then. you should also change the `${COMPILER}` below.

Reply:
Yeah exactly, it didnt't work before hand, so the clang container always had the gcc dunebase in it